### PR TITLE
refactor: Move various consts to `constants.rs`

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Move
- list of core Northstar mods
- list of blacklisted mods
- list of Titanfall2 Origin game IDs

to `constants.rs`

This should cover all remaining constants.

### CODE REVIEW

File: src-tauri/src/mod_management/mod.rs [hunk 1]
```diff
@@ -12,8 +13,6 @@ use app::GameInstall;
 
 use json5;
 
-use crate::northstar::CORE_MODS;
-
 pub const BLACKLISTED_MODS: [&str; 3] = [
```
Comment (Alystrasz) [inline src-tauri/src/mod_management/mod.rs:16]: Would it make sense to move those to constants as well?

Comment (GeckoEidechse) [reply src-tauri/src/mod_management/mod.rs]: Done with 3d62be62e79f0f079eaa73e4072836da163da020 and also found one more and moved with 0eab7afe6ce0e2e14674b4a364faa52fdd75c37c ^^

File: src-tauri/src/constants.rs [hunk 2]
```diff
@@ -7,3 +7,21 @@ pub const MASTER_SERVER_URL: &str = "https://northstar.tf";
 
 // server list endpoint
 pub const SERVER_BROWSER_ENDPOINT: &str = "/client/servers";
+
+// List of core Northstar mods
+pub const CORE_MODS: [&str; 3] = [
```
Comment (Alystrasz) [inline src-tauri/src/constants.rs:12]: I don't remember the context for #120, but this PR is the occasion to solve it :)

**Review by Alystrasz — CHANGES_REQUESTED** [review]

File: src-tauri/src/constants.rs (hunk 2 above)
Comment (GeckoEidechse) [reply src-tauri/src/constants.rs:12]: The context was that we declare core mods twice, once in frontend, once in backend which is just unnecessary duplication. As one cannot just do like 

```rust
#[ts(export)]
pub const BLACKLISTED_MODS: [&str; 3] = [
    "northstar-Northstar",
    "northstar-NorthstarReleaseCandidate",
    "ebkr-r2modman",
];
```

I'll have to add some function that frontend could call to get the list of core mods or other other constants. Will look into it but that shouldn't block this PR here, as this PR is basically a prerequisite for the other future one ^^

Comment (GeckoEidechse) [reply src-tauri/src/constants.rs:12]: Sorta addressed in #168

**Review by Alystrasz — APPROVED** [review]
LGTM.
Confirmed working on Windows.

**Pull request merged**